## Berikut adalah projek 3
Soal-soal terkahir dari kami sebagai pengajar Data Science Indonesia Regional Jawa Timur

### <i>Dataset</i>

In [1]:
from pandas import read_csv

data = read_csv("data.csv")
data

CYT   NUC   MIT   ME3   ME2  ME1  EXC   VAC   POX
0     ADT1_YEAST  0.58  0.61  0.47  0.13  0.5  0.0  0.48  0.22
1     ADT2_YEAST  0.43  0.67  0.48  0.27  0.5  0.0  0.53  0.22
2     ADT3_YEAST  0.64  0.62  0.49  0.15  0.5  0.0  0.53  0.22
3     AAR2_YEAST  0.58  0.44  0.57  0.13  0.5  0.0  0.54  0.22
4     AATM_YEAST  0.42  0.44  0.48  0.54  0.5  0.0  0.48  0.22
...          ...   ...   ...   ...   ...  ...  ...   ...   ...
1479  YUR1_YEAST  0.81  0.62  0.43  0.17  0.5  0.0  0.53  0.22
1480  ZIP1_YEAST  0.47  0.43  0.61  0.40  0.5  0.0  0.48  0.47
1481  ZNRP_YEAST  0.67  0.57  0.36  0.19  0.5  0.0  0.56  0.22
1482  ZUO1_YEAST  0.43  0.40  0.60  0.16  0.5  0.0  0.53  0.39
1483  G6PD_YEAST  0.65  0.54  0.54  0.13  0.5  0.0  0.53  0.22

[1484 rows x 9 columns]

Data di atas digunakan untuk menyelesaikan soal nomor satu dan dua. Adapun keterangan pada <i>dataset</i> dapat dilihat pada file yang bernama <b>Info.names</b>. <i>Dataset</i> di atas memiliki 1484 baris dan 9 kolom. Mengingat soal nomor satu dan dua bukan ditekankan pada <i>Exploratory Data Analysis</i> maka untuk menjawab soal nomor satu dan dua tidak perlu menunjukan visualisasi dari data di atas.

## 1. <i>Unsupervised Learning</i>

Berdasarkan data yang tersedia, terapkan metode klasterisasi guna mengelompokkan tiap baris pada kelompok tertentu. Sehingga hasil akhir yang diminta adalah 1 kolom tambahan yang merupakan penentu baris tersebut adalah kategori/klaster yang mana. Untuk menjawab soal ini, silahkan simak materi pertemuan ke sepuluh yang membahas <i>predictive analytics</i> (<i>unsupervised learning/clustering</i>).

## 2. <i>Supervised Learning</i>

Setelah mendapatkan kategori untuk tiap baris pada data di atas, maka latih sebuah algoritma klasifikasi pada <i>dataset</i> yang barusan kalian olah untuk dapat memprediksi masukan baru. Untuk menjawab soal ini, silahkan simak materi pertemuan ke sembilan yang membahas <i>predictive analytics</i> (<i>supervised learning</i>).

## 3. <i>Data Storytelling</i>

Carilah <i>dataset</i> yang hendak kalian gunakan untuk "bercerita" terkait penemuan wawasan yang kalian coba usahakan. Minimal 3 visualisasi yang menarik.